login-workflow/example/src/index.tsx

```diff
@@ -9,6 +9,7 @@ import { blueThemes } from '@brightlayer-ui/react-themes';
 import '@brightlayer-ui/react-themes/open-sans';
 import './index.css';
 import { createRoot } from 'react-dom/client';
+import type {} from '@mui/material/themeCssVarsAugmentation';
```

Comment:
add direction to theme to fix RTL issue

Reply:
We don't have RTL functionality in the example app.